### DISCUSSION TITLE
No option to search in online documentation

### DISCUSSION THREAD

**Discussion by aniketgm** [GothenburgBitFactory/taskwarrior#3277] [General]
The online helps seems to be missing the Search feature. The documentation is nice and big, but there is no option to search anything within the documentation -- https://taskwarrior.org/docs/
I just wanted to find how to set the `_forcecolor` property ? How do I know under which heading is it in the doc ? I have to go through each heading/subheading.

After a bit of browsing through docs I found it. But it still doesn't mentioned how do I set it in the taskrc file.
I also found something called `report.customnxt`. Is `rc` and `report.customnxt` same thing, ( because of the `r`, `c` ) ? It's a bit confusing.

**Comment by djmitche** [comment]
Taskwarrior is a CLI application, so the authoritative documentation is in the manpages. You can find information on `_forcecolor` in taskrc(5). That manpage also describes how to set configuration variables. Typically that's in the `.taskrc` file, but, to quote the manpage:

```
       Individual options can be overridden by using the rc.<name>: attribute when running task:

              $ task rc.<name>:<value> ...

       or

              $ task rc.<name>=<value> ...
```

I think `rc` is short for `taskrc`.

On an personal note, opening a conversation by criticizing the people from whom you're looking for help is not a very effective strategy. This is an open-source project, and like all open source projects has many more people demanding things of it than people doing the work. It doesn't feel great when someone says "..is not good" about something that I work on in my spare time. I've tried to look past this insult here and answer your question, but I invite you to consider other ways of approaching open-source projects in the future.

Finally, this _is_ an open source project, and perhaps the reason the documentation is "so bad" is because _you_ haven't made it better. Once you're more familiar with Taskwarrior, we'd love to hear your plans for how to make the documentation better, and review the PRs you submit.

**Reply by aniketgm** [reply]
> I've tried to look past this insult here and answer your question

I'm sorry if it felt like that. However, I stand with my statement "..is not good". I'm not happy with the online help pages.

> Finally, this is an open source project, and perhaps the reason the documentation is "so bad" is because you haven't made it better.

Oh, I definitely plan to make it better. Thing is, I recently started with taskwarrior. And as for anyone starting anything, they look for documentation online. Although, I have started browsing manpages now instead of online, which also has a way to quickly search something and move on ahead.
